vaccine_feed_ingest/runners/ga/dph/parse.py

```diff
@@ -2,29 +2,98 @@
 
 import json
 import pathlib
+import re
 import sys
+from typing import List
 
 from bs4 import BeautifulSoup
+from fetch import location_file_name_for_url
 
 
-def main():
-    output_dir = pathlib.Path(sys.argv[1])
-    input_dir = pathlib.Path(sys.argv[2])
+def parse_location(input_file: pathlib.Path) -> dict:
+    result: dict = {"phone-numbers": [], "contact-links": []}
+    with input_file.open() as f:
+        doc = BeautifulSoup(f, "html.parser")
+    if doc is None:
+        raise Exception("failed to set up beautiful soup")
+    main_el = doc.find(id="main-content")
+    if main_el is None:
+        raise Exception("main content block has changed classes")
```

Comment:
Thank you for adding errors like these which will help us maintain this over time.